src/font_renderer/ct_font_renderer.cpp

```diff
@@ -0,0 +1,140 @@
+
+#if DPFB_USE_CORETEXT
+
+#include "font_renderer/font_renderer.h"
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <CoreGraphics/CoreGraphics.h>
+#include <CoreText/CoreText.h>
+
+
+
+class CtFontRenderer : public FontRenderer {
+public:
+    explicit CtFontRenderer(const FontRendererArgs& args);
+
+    FontMetrics getFontMetrics() const override;
+
+    GlyphIndex getGlyphIndex(char32_t cp) const override;
+    GlyphMetrics getGlyphMetrics(GlyphIndex glyphIdx) const override;
+    void renderGlyph(GlyphIndex glyphIdx, Image& image) const override;
+    virtual ~CtFontRenderer() override;
+private:
+    CTFontRef font;
+};
+
+
+CtFontRenderer::CtFontRenderer(const FontRendererArgs& args)
+{
+    CFStringRef fontName = CFStringCreateWithCString(NULL, (const char*)args.data, kCFStringEncodingUTF8);
+    font = CTFontCreateWithName(fontName, args.pxSize, NULL);
```

Comment:
I'm not familiar with Core Text, but is it OK that you pass the whole font data as a name?

Reply:
Hm, I forgot to look at this, though it worked for the font which I needed.

Reply:
LOL, it just used Helvetica as default font, because there is no font named `ttcf`, and I only needed Helvetica. Fixed with c7b87610b5dbb660fd4e7da654da4199526fb3f7, tested on Times and Helvetica.